src/services/PopularCitiesServise.js

```diff
@@ -0,0 +1,35 @@
+import instance from '../utilities/axios/axios';
+
+const DEFAULT_PAGE = 1;
+const DEFAULT_PAGE_SIZE = 10;
+const DEFAULT_SORT_ORDER = 'POPULARITY_DESC';
+const DEFAULT_ENDPOINT = '/cities';
+
+async function fetchPopularCities({
+  endpoint = DEFAULT_ENDPOINT,
+  page = DEFAULT_PAGE,
+  pageSize = DEFAULT_PAGE_SIZE,
+  sortOrder = DEFAULT_SORT_ORDER,
+  onlyWithGuides = false,
+} = {}) {
+  const params = {
+    page,
+    pageSize,
+    sortOrder,
+    onlyWithGuides,
+  };
+
+  try {
+    const response = await instance.get(endpoint, { params });
+    if (response.data && response.data.data.items) {
+      return response.data.data.items;
+    } else {
+      throw new Error('No data found');
```

Comment:
Be more specific, error message is too general

```suggestion
      throw new Error('Unable to retrieve popular cities');
```

Reply:
When I caught an error, in order to understand at what stage of the request I was unable to receive data, I did console.log at each stage and wrote down all the errors briefly. You're right, I need to rename and add a couple more.